managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -1089,16 +1152,16 @@ private void checkForNewEntries(OpReadEntry op, ReadEntriesCallback callback, Ob
                 }
                 // Try to cancel the notification request
                 if (WAITING_READ_OP_UPDATER.compareAndSet(this, op, null)) {
+                    ledger.removeWaitingCursor(this);
                     if (log.isDebugEnabled()) {
                         log.debug("[{}] [{}] Cancelled notification and scheduled read at {}", ledger.getName(),
-                                name, op.readPosition);
+                            name, op.readPosition);
                     }
                     PENDING_READ_OPS_UPDATER.incrementAndGet(this);
                     ledger.asyncReadEntries(op);
                 } else {
-                    if (log.isDebugEnabled()) {
-                        log.debug("[{}] [{}] notification was already cancelled", ledger.getName(), name);
-                    }
+                    log.info("[{}] [{}] notification that new entries added was already be called, skipped the curren"
+                        + "t new entry checking", ledger.getName(), name);
```

Comment:
Could you avoid splitting a word  (`"current"`) into two string literals?

Reply:
Addressed